### ISSUE TITLE
sys-net can't establish ethernet connetcion after dom0 update

### ISSUE THREAD

**Issue by crazyqubes** [QubesOS/qubes-issues#3546]
#### Qubes OS version:

R3.2

#### Affected TemplateVMs:

fedora-23

---

### Steps to reproduce the behavior:

update dom0 after fresh installation of qubes R3.2

### Expected behavior:

update dom0 and still got an ethernet connection

### Actual behavior:

update dom0 and got no ethernet connection 

### General notes:

---

**Comment by andrewdavidwong** [comment]
Please provide more specific information, including the version numbers of the packages you updated (see [Reporting Bugs](https://www.qubes-os.org/doc/reporting-bugs/)).

**Comment by crazyqubes** [comment]
Hi Andrew, 

i made a list of all packages including the version numbers I updated. I hope that´s what you need. 

Thanks for your work!


![github 1](https://user-images.githubusercontent.com/36197182/35918733-c5cfb906-0c0a-11e8-94b7-2f2e9d09e57c.jpg)

**Comment by marmarek** [comment]
Is sys-net starting at all? Do you see network manager applet, what devices are listed there?

**Comment by crazyqubes** [comment]
Hey @marmarek,

yes sys-net is starting also the network manager applet. There is still my "realtek" ethernet device listed, but as unwired.

**Comment by marmarek** [comment]
Does anything change if you disconnect and connect the cable? Try opening terminal in sys-net and check kernel messages (`sudo dmesg`) - look for error related to your device. There should be also driver name (prefix of those errors) - check if you have this module loaded (`lsmod` command), then unload it (`sudo modprobe -r ...`) and load again (`sudo modprobe ...`). If that helps, add the module to `/rw/config/suspend-module-blacklist` file and it should be fixed.

**Comment by crazyqubes** [comment]
Nothing change when i disconnect and connect the cable. 
Like you assumed there is an error related to my device (sudo dmesg).

```
r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
r8169 0000:00:00.0: Xen PCI mapped GSI32 to IRQ22
r8169 0000:00:00.0: pci frontend enable msi failed for dev 0:0
r8169 0000:00:00.0: Xen PCI frontend error: -22!

```

I unloaded and loaded module r8169, nothing happened, unplugged/plugged in the ethernet cable, nothing happened.

To be sure i unloaded/ loaded the module xen_pcifront too, same result as above with module r8169.

**Comment by marmarek** [comment]
Oh, that's interesting. @HW42 can you help? Looks like enabling MSI works only once.

**Comment by marmarek** [comment]
In practice this looks like a duplicate of #3217
